types/providers.d.ts

```diff
@@ -99,6 +100,7 @@ export type OAuthProviderType =
   | "WordPress"
   | "WorkOS"
   | "Yandex"
+  | "Yuque"
```

Comment:
Can you add [type tests](https://github.com/nextauthjs/next-auth/blob/788d7dc41f8767a4659136475189d6d420295855/types/tests/providers.test.ts#L259) for **Yuque** as well? 😊